kythe/typescript/indexer.ts

```diff
@@ -782,9 +787,11 @@ class Visitor {
   }
 
   /** newAnchor emits a new anchor entry that covers a TypeScript node. */
-  newAnchor(node: ts.Node, start = node.getStart(), end = node.end): VName {
+  newAnchor(node: ts.Node, start = node.getStart(), end = node.end, tag = ''):
+      VName {
     const name = Object.assign(
-        {...this.kFile}, {signature: `@${start}:${end}`, language: LANGUAGE});
+        {...this.kFile},
+        {signature: `@${tag}${start}:${end}`, language: LANGUAGE});
```

Comment:
What's the purpose of distinguishing the anchor defining the two type param nodes?

Reply:
I wanted to keep the two subgraphs entirely parallel. Does this make any difference after denormalization? ISTR exactly overlapping anchors get special treatment somewhere.

Reply:
It'll be fine in post-processing.  Only `imputes` edges matter when it comes to overlapping anchor spans.